docs/templates/index.md

```diff
@@ -0,0 +1,199 @@
+# Templates 
+
+A template is a collection of configuration parameters that is used to launch a Triton instance. Templates are immutable,
+therefore, once they are created, they cannot be changed, therefore a new template needs created. A template is made up
+as follows:
+
+| Field             | Type              |  Description                                                                  |
+|----------         |-------------      |------                                                                         |
+| ID                | string            | The UUID of the Template.                                                     |
```

Comment:
Across this table the following works are capitalized in the description field that should not be:
+ Template
+ Package
+ Firewall
+ AccountID (should be account ID in the description)

Those items are not proper nouns and should be lowercase.